From patches Mon Sep 17 00:00:00 2001
From: liguang <lig.fnst@cn.fujitsu.com>
To: Michael Tokarev <mjt@tls.msk.ru>, qemu-devel <qemu-devel@nongnu.org>,
 liguang <lig.fnst@cn.fujitsu.com>
Cc: Alexander Graf <agraf@suse.de>, qemu-devel <qemu-devel@nongnu.org>,
 qemu-trivial@nongnu.org, QEMU Trivial <qemu-trivial@nongnu.org>,
 Alexey Kardashevskiy <aik@ozlabs.ru>, liguang <lig.fnst@cn.fujitsu.com>,
 Peter Maydell <peter.maydell@linaro.org>
Date: Tue, 30 Apr 2013 10:59:50 +0800
Message-id: 1367290790-19159-1-git-send-email-lig.fnst@cn.fujitsu.com
Subject: [PATCH 1/1] translate: remove redundantly included qemu/timer.h

Signed-off-by: liguang <lig.fnst@cn.fujitsu.com>
Reviewed-by: Peter Maydell <peter.maydell@linaro.org>
---
 translate-all.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/translate-all.c b/translate-all.c
index da93608..d04a116 100644
--- a/translate-all.c
+++ b/translate-all.c
@@ -55,7 +55,6 @@
 #else
 #include "exec/address-spaces.h"
 #endif
-#include "qemu/timer.h"
 
 #include "exec/cputlb.h"
 #include "translate-all.h"
-- 
1.7.2.5


